Aquality.Selenium/src/Aquality.Selenium/Resources/settings.json

```diff
@@ -67,6 +67,21 @@
       },
       "startArguments": []
     },
+    "edgechromium": {
+      "webDriverVersion": "82.0.458.0",
```

Comment:
why do not we use here Latest as for others?

Reply:
Latest version doesn't work for me (I have installed 83 version), so I not able to check it and that is why I am not sure we need to set Latest here